ports/stm32/usbd_cdc_interface.c

```diff
@@ -48,6 +48,10 @@
 
 #if MICROPY_HW_ENABLE_USB
 
+#if MICROPY_HW_USB_CDC_1200BPS_TOUCH
+#include "powerctrl.h"
```

Comment:
Cleaner to just unconditionally include this (and include it alongside the other includes above).